jenkins-master/jobs/mozilla-aurora_functional/config.xml

```diff
@@ -195,7 +174,6 @@ ${FAILED_TESTS}
       <timeoutType>absolute</timeoutType>
       <timeoutMinutesElasticDefault>3</timeoutMinutesElasticDefault>
     </hudson.plugins.build__timeout.BuildTimeoutWrapper>
-    <hudson.plugins.timestamper.TimestamperBuildWrapper plugin="timestamper@1.5.14"/>
```

Comment:
The mozharness logger already outputs timestamps. So I don't see the need to have this information twice and stealing visible space per line.